### ISSUE TITLE
Swagger UI doesn't show a textarea for the body in the "Try it out" functionality

### ISSUE THREAD

**Issue by DirkGuijt** [channelcat/sanic-openapi#57]
I'm on version 0.4.0 and my swagger UI doesn't seem to show a body textarea where I can modify the body when trying out an API endpoint.

You can resproduces this (hopefully) by copying the `/user ` endpoint from the readme file:

```
@app.post("/user")
@doc.summary("Creates a user")
@doc.consumes({"user": { "name": str }}, location="body")
async def create_user(request):
    ...
```

When you click "Try it out" on the `/user` endpoint on the Swagger UI, you can't modify the body. Because of this, you can't even execute the call if you make the input a required field.

**Comment by DirkGuijt** [comment]
this was due to an error in the generated openapi spec.json, fixed it in pull request #58

**Comment by mt-rahul** [comment]
> this was due to an error in the generated openapi spec.json, fixed it in pull request #58

Are the fixes yet to be merged with the master branch? I am still facing the same issue.
